qiita_pet/handlers/rest/study_samples.py

```diff
@@ -172,12 +172,7 @@ def patch(self, study_id):
             if set(data.columns).issubset(categories):
```

Comment:
Should this and the previous if be merged?

Reply:
Good catch!